File: EXERCICIOS/Lista 2/Questao1.py
import math
tot = 0
medida = [0] * 10
for i in range(10):
    medida[i] = float(input(f"medida {i}: "))
    tot += medida[i]
resultado = tot / len(medida)
varianc = sum((x - resultado) ** 2 for x in medida) / len(medida)
media = math.sqrt(varianc)
print("Média:", resultado)
print("DesvPad:", media)

if media > resultado * 0.1:
    print("O multimetro está com defeito")
else:
    print("O multimetro está aferido corretamente")

File: EXERCICIOS/Lista 1/Questao1.java
import java.util.Scanner;

public class Questao1 {
    public static void main(String[] args) {
        double tot = 0;
        final double[] medida = new double[10];

        Scanner scanner = new Scanner(System.in);

        for (int i = 0; i < medida.length; i++) {
            System.out.printf("medida %d: ", i);
            double read = scanner.nextDouble();

            medida[i] = read;
            tot += read;
        }

        final double resultado = tot / medida.length;

        double varianc = 0;
        for (double v : medida)
            varianc += Math.pow(v - resultado, 2);
        varianc /= medida.length;

        // Standard deviation
        double media = Math.sqrt(varianc);

        System.out.println("Média: " + resultado);
        System.out.println("DesvPad: " + media);

        if (media > resultado * 0.1) {
            System.out.println("O multimetro está com defeito");
        } else {
            System.out.println("O multimetro está aferido corretamente");
        }
    }
}

